packages/wallet/src/redux/protocols/funding/player-b/reducer.ts

```diff
@@ -56,10 +55,7 @@ export function fundingReducer(
   sharedData: SharedData,
   action: actions.FundingAction | EmbeddedAction,
 ): ProtocolStateWithSharedData<states.FundingState> {
-  if (
-    isAdvanceChannelAction(action) &&
-    action.protocolLocator === ADVANCE_CHANNEL_PROTOCOL_LOCATOR
-  ) {
+  if (isAdvanceChannelAction(action)) {
```

Comment:
I think we need to check the `protocolLocator` in addition to checking `isAdvanceChannelAction` since an `AdvanceChannelAction`  could be meant this protocol or a sub protocol. Without checking the `protocolLocator`we'll prevent any `AdvanceChannelAction`s from being sent to a sub protocol.